### PR TITLE
Fix bugs in 5.0 docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Fix some bugs.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by sticnarf** [comment]
And there are some other wording changes: https://github.com/pingcap/docs-cn/pull/5181#pullrequestreview-559758890

@scsldb also suggests "本特性与 TiDB Binlog 不兼容，开启 TiDB Binlog 时本配置将不生效。" should also be a warning.

**Comment by TomShawn** [comment]
> And there are some other wording changes: [#5181 (review)](https://github.com/pingcap/docs-cn/pull/5181#pullrequestreview-559758890)
> 
> @scsldb also suggests "本特性与 TiDB Binlog 不兼容，开启 TiDB Binlog 时本配置将不生效。" should also be a warning.

Done

**Review by sticnarf — APPROVED** [review]
LGTM

File: system-variables.md [hunk 1]
```diff
@@ -1000,7 +1000,7 @@ explain select * from t where age=5;
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
 
-### `tidb_track_aggregate_memory_usage` <!-- 从 v5.0.0-rc 版本开始引入 -->
+### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:1003]:
```suggestion
### `tidb_track_aggregate_memory_usage` <!-- 从 v5.0.0-rc 版本开始引入 -->
```

**Review by yikeke — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -1018,12 +1018,13 @@ explain select * from t where age=5;
 >
 > + 暂时与 [TiCDC](/ticdc/ticdc-overview.md) 不兼容，可能导致 TiCDC 运行不正常。
 > + 暂时与 [Compaction Filter](/tikv-configuration-file.md#enable-compaction-filter) 不兼容，共同使用时有小概率发生写丢失。
+> + 本特性与 TiDB Binlog 不兼容，开启 TiDB Binlog 时本配置将不生效。
```
Comment (TomShawn) [inline system-variables.md:1021]:
```suggestion
> + 本特性与 TiDB Binlog 不兼容，开启 TiDB Binlog 时本配置将不生效。 
```

**Pull request merged**